src/api/v2/ResourceGroupsExtensionManager.ts

```diff
@@ -27,4 +40,37 @@ export class ResourceGroupsExtensionManager {
             }
         }
     }
+
+    async activateApplicationResourceProviders(): Promise<void> {
+        const inActiveResourceContributors =
+            vscode.extensions.all
+                .map(extension => ({ extension, resources: (extension.packageJSON as ExtensionPackage)?.contributes?.['x-azResourcesV2']?.application?.resources ?? [] }))
```

Comment:
A helper method to get `(extension.packageJSON as ExtensionPackage)?.contributes?.['x-azResourcesV2']` would be nice. It's not fun to type and/or look at 😄

Reply:
Consolidated into a common function.